src/components/buttons/Button.spec.jsx

```diff
@@ -16,13 +16,6 @@ test('type', () => {
   expect(button.hasClass(`sg-button--${buttonType}`)).toEqual(true);
 });
 
-test('button with href', () => {
```

Comment:
let's discuss about our plans removing enzyme entirely. As far as I remember we've been talking about possibly moving both a11y and classic tests to one `.spec` but let's bring back this topic and make a decision.

cc @domidomi @clxandstuff